docs/2.7.4/docs/canton/usermanual/upgrading.rst

```diff
@@ -280,6 +288,13 @@ Specific error changes are as follows:
 
 * The ``ContractKeyNotVisible`` error (previously encapsulated by ``GenericInterpretationError``) is now transformed into a ``ContractKeyNotFound`` to avoid information leaking.
 
+Upgrade to Release 2.6
+^^^^^^^^^^^^^^^^^^^^^^
+Version 2.6 will change the database schema used. Therefore, you will have to perform the
+database migration steps. Please note that depending on the size of the database, this operation can take many hours. 
+Ensure that before starting your nodes, you vacuum your database to avoid long startup times. The participant 
+node will refuse to startup due to initially extremely long database response times otherwise.
```

Comment:
```suggestion
Ensure that before starting your nodes, you vacuum your database to avoid long startup times. Otherwise, the participant 
node will refuse to start due to extremely long initial database response times.
```

Reply:
Closing because Carrie's comments supersede mine